### DISCUSSION TITLE
Push a new version to elpa

### DISCUSSION THREAD

**Discussion by paaguti** [company-mode/company-mode#1463] [Ideas]
Hi,
I see that the github version disables ispell suggestions quite nicely. Would it be possible to push the current state to elpa? I feel  ispell autocompletions popping up when I write documents too intrusive and distractive. I had to suggest to a couple of people (with challenges that make it difficult for them to concentrate) to disable company mode completely for LaTeX and org. They loose LSP completions, which on the other hand, they do need.

Thanks a ton, /PA

**Comment by dgutov** [comment]
Hi!

I'm putting the finishing touches, so it might be a little longer, sorry.

Would you like a snippet to use in the meantime? Or you could also install from MELPA.
